docs/admin/telemetry.rst

```diff
@@ -0,0 +1,37 @@
+.. _telemetry:
+
+Telemetry
+=========
+
+To help to improve Artemis, we collect some data when the application starts.
+This feature can be disabled by setting `telemetry.enabled` in the `application-prod.yml` to `false`.
+When this is set to false, no data is sent to Artemis.
```

Comment:
```suggestion
When this is set to false, no data is sent to the Artemis maintainer team.
```
Something like this. Just Artemis is ambiguous otherwise since every operator has an Artemis instance.